modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectRelationshipLocalServiceImpl.java

```diff
@@ -1984,10 +1984,12 @@ private void _validateNameObjectRelationshipName(
 				"name."));
 	}
 
-	private void _validateObjectEntryId(
+	private ObjectEntry _validateObjectEntryId(
```

Comment:
The method name `_validateObjectEntryId` doesn’t reflect the fact that it now returns an `ObjectEntry`.
Can you rename it to something that better represents the action it performs?

Reply:
`getObjectEntryGroupId`